packages/components/stories/Datalist.js

```diff
@@ -23,6 +23,19 @@ const propsMultiSection = {
 			suggestions: [{ name: 'My foobar', value: 'foobar', description: 'foobar description' }],
 		},
 		{ title: 'cat 4', suggestions: [{ name: 'My lol', value: 'lol' }] },
+		{
+			title: 'cat 1',
```

Comment:
I think changing this storybook is irrelevant here.
And you add the same existing values, which is not a supported case :D .
I think you can revert it

Reply:
Forgot i pushed it 🤦‍♂️